compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/convert_to_dma.mlir

```diff
@@ -80,6 +80,28 @@ func.func @subview_pack() {
   return
 }
 
+// -----
+// CHECK-LABEL: @collapsing_subview_pack
+// CHECK: %[[SRC_LOFI:.*]] = amdaie.logicalobjectfifo.from_memref {{.*}} !amdaie.logicalobjectfifo<memref<12x5x2x10x6x8xf32>>
+// CHECK: %[[DST_LOFI:.*]] = amdaie.logicalobjectfifo.from_memref {{.*}} !amdaie.logicalobjectfifo<memref<2x2x3x3xf32, 1>>
+// CHECK: amdaie.dma_cpy_nd
+// CHECK-SAME: %[[DST_LOFI]][0, 0, 0, 0] [2, 2, 3, 3] [18, 9, 3, 1]
+// CHECK-SAME: %[[SRC_LOFI]][0, 0, 0, 0] [2, 2, 3, 3] [14400, 480, 8, 4800]
```

Comment:
How does the stride [14400, 480, 8, 4800] calculated?

Reply:
- The strides on `%sbv` is `[4800, 480, 8]`. 
- Packing goes from `6x2x3` to `2x2x3x3` by moving dim 1 to the end (applying permutation [0, 2, 3, 1])

This is effectively 2 steps, a shape expansion and then a permutation/dimShuffle: 

1) Expansion from `6x2x3` to `2x3x2x3` , the strides on this are [4800*3, 4800, 480, 80]. 

2) The permutation from `2x3x2x3` to `2x2x3x3` by moving dimension 1 to the end. This permutation is applied to the strides, giving [4800, 480, 80, 4800*3].

Reply:
* last line typo: giving [4800*3, 480, 80, 4800]